blocks/procedures.js

```diff
@@ -126,7 +126,7 @@ Blockly.Blocks['procedures_defnoreturn'] = {
   /**
    * Create XML to represent the argument inputs.
    * @param {=boolean} opt_paramIds If true include the IDs of the parameter
-   *     quarks.  Used by Blockly.Procedures.mutateCallers for reconnection.
+   *    quarks.  Used by Blockly.Procedures.mutateCallers for reconnection.
```

Comment:
Because it's four from the * and I couldn't tell where the four was supposed to start.

But now I know, and knowledge is power.